changelog.d/9003.misc

```diff
@@ -0,0 +1 @@
+Remove fts4 shadow tables from schema dumps.
```

Comment:
Could you update this to describe what this means for the user?

"Fix 'object name reserved for internal use' with recent versions of SQLite." or so?

Reply:
Added 🚀